config/locales/ar.yml

```diff
@@ -1703,12 +1703,6 @@ ar:
   card_has_been_removed: "تمت إزالة بطاقتك (الرقم: %{number})"
   card_could_not_be_removed: عذرًا ، تعذرت إزالة البطاقة
   invalid_credit_card: "بطاقة الائتمان غير صالحة"
-  ie_warning_headline: "متصفحك غير محدث :-("
-  ie_warning_text: "للحصول على أفضل تجربة لشبكة الغذاء المفتوح ، نوصي بشدة بترقية متصفحك:"
-  ie_warning_chrome: تحميل متصفح كروم
-  ie_warning_firefox: تحميل متصفح فايرفوكس
-  ie_warning_ie: ترقية Internet Explorer
-  ie_warning_other: "لا يمكنك ترقية متصفحك؟ جرب شبكة الغذاء المفتوح على هاتفك الذكي :-)"
```

Comment:
We should only change en.yml. The others will be updated by Transifex and we don't want merge conflicts.

Reply:
Thanks for confirming this, good to know.